pkgs/development/python-modules/wxPython/4.0.nix

```diff
@@ -22,14 +37,43 @@ buildPythonPackage rec {
     sha256 = "35cc8ae9dd5246e2c9861bb796026bbcb9fb083e4d49650f776622171ecdab37";
   };
 
-  nativeBuildInputs = [
-    pkgconfig
-  ];
+  doCheck = false;
 
-  buildInputs = [
-    gtk3 libjpeg libtiff SDL gst-plugins-base libnotify freeglut xorg.libSM
-    which
-  ];
+  nativeBuildInputs = [ pkgconfig which doxygen wxGTK libX11 ];
```

Comment:
Like I said before, `wxGTK` and `libX11` are libraries, so let's move them to `buildInputs` if they are needed.

Reply:
@veprbl if I do this, the build fails *shrug*

Reply:
The build succeeds if I remove `libX11`. Moving `wxGTK`, indeed, fails because it can't find "wx-config", which I don't understand ATM.

Reply:
ah thanks.

Reply:
Does this still matter? I'm going back to take another shot at enabling tests and I might look into this or add it as an open problem.

Reply:
It is probably okay now
https://github.com/NixOS/nixpkgs/blob/b1bffdf67fd415fdae81c470271959830eba89f7/pkgs/development/python-modules/wxPython/4.0.nix#L40-L48